.travis.yml

```diff
@@ -9,6 +9,8 @@ install:
   - go get -v github.com/mattn/goveralls
 
 script:
+  # This unencryptes envvars.bash.enc into envvars.bash so that we can run the tests
+  - openssl aes-256-cbc -K $encrypted_263604b5b445_key -iv $encrypted_263604b5b445_iv -in envvars.bash.enc -out envvars.bash -d
```

Comment:
Thanks @oxtoacart!

@myleshorton I think that link ultimately refers to a private repository (I'm getting 404 while trying to access envvars.bash), could you add me there?